.gitignore

```diff
@@ -41,3 +41,4 @@ test/client-old/spec/mocks/translations.js
 .elasticbeanstalk/*
 !.elasticbeanstalk/*.cfg.yml
 !.elasticbeanstalk/*.global.yml
+/nbproject/private/
```

Comment:
Agreed - this was not intended to make the pull request, I'll remove it from my branch.